src/App.tsx

```diff
@@ -1,7 +1,70 @@
-import React from "react";
-
-function App() {
-  return <div className="App">Hey :)</div>;
-}
+import * as React from "react";
+import Map from "./containers/Map";
+import Sidebar from "./containers/Sidebar";
+import SidebarControl from "./containers/SidebarControl";
+import { Box } from "@material-ui/core";
+import { useWindowDimensions } from "./utils";
+import "./App.css";
+export const AppContext: any = React.createContext({});
 
+const App: React.FC = () => {
+  const { width } = useWindowDimensions();
+  const [openSidebar, setOpenSidebar] = React.useState(true);
+  const [activeSidebarItem, setActiveSidebarItem] = React.useState(0);
+  const [view] = React.useState({
+    longitude: 12.568337,
+    latitude: 55.676098,
+    zoom: 5.5,
+  });
+  const handleToggleDrawer = () => {
+    setOpenSidebar(!openSidebar);
+  };
+  return (
+    <AppContext.Provider
+      value={{
```

Comment:
also, here would be nice to have the `actions` and `state` alphabetically ordered, having a recognizable pattern you can find them easily